src/lib.rs

```diff
@@ -13,5 +13,13 @@
 pub mod gl;
 
 mod ffi {
+    include!(concat!(env!("OUT_DIR"), "/gl_and_gles_bindings.rs"));
+}
+
+mod ffi_gl {
     include!(concat!(env!("OUT_DIR"), "/gl_bindings.rs"));
 }
+
+mod ffi_gles {
+    include!(concat!(env!("OUT_DIR"), "/gles_bindings.rs"));
+}
```

Comment:
The pull request uses current gl_generator. It could not generate "GL+GLES constants", GL FFI and GLES ffi combination. Then 3 versions are used to generate them. servo/gleam#108 is for avoiding the 3 versions.